rrule/rrule.d.ts

```diff
@@ -158,4 +158,27 @@ declare namespace RRule {
 
     function fromString(value: string): RRule;
 
+    class RRuleSet extends RRule {
```

Comment:
I don't think the RRuleSet is exposed in version 2.1.0. The current master does have it although it is not released yet.